app/models/manageiq/providers/azure/cloud_manager/cloud_database.rb

```diff
@@ -18,32 +35,47 @@ def self.params_for_create
           :label        => _('Database Type'),
           :includeEmpty => true,
           :isRequired   => true,
-          :validate   => [{:type => 'required'}],
+          :validate     => [{:type => 'required'}],
           :options      => ['MySQL', 'SQL', 'MariaDB', 'PostgreSQL'].map do |db|
             {
               :label => db,
-              :value => db
+              :value => db,
             }
           end,
         },
-        {
-          :component => 'text-field',
-          :id        => 'server',
-          :name      => 'server',
-          :label     => _('Database Server Name'),
-        },
-        {
-          :component => 'text-field',
-          :id        => 'resource_group',
-          :name      => 'resource_group',
-          :label     => _('Resource Group'),
-        },
+        
+        # {
+        #   ## TODO this condition causes error + cannot set the values
+        #   ## of SqlServerService (database type) and test-group (resource group) dynamically.
+        #   ## This whole select will need to be moved elsewhere.
+        #
+        #   :component => 'select',
+        #   :id        => 'server',
+        #   :name      => 'server',
+        #   :label     => _('Database Server Name'),
+        #   :includeEmpty => true,
+        #   :isRequired => true,
+        #   :validate   => [{:type => 'required'}],
+          
+        #   :options      => ems.connect(:service => 'SqlServerService').list('test-group') do |svr|
+        #     {
+        #       :label => svr[:name],
+        #       :value => svr[:name],
+        #     }
+        #   end,
+        #   :condition => {
+        #     :and => [
+        #       {when: 'resource_group', isNotEmpty: true,},
+        #       {when: 'database', isNotEmpty: true,},
+        #     ]
+        #   },
+        # },
```

Comment:
Don't think we should to leave this code commented out here since the `ems.connect()` is not valid, if anything a TODO or NOTE above the database server name selection above that we should limit by the selected resource_group in the future